stetho/src/main/java/com/facebook/stetho/inspector/MethodDispatcher.java

```diff
@@ -63,6 +64,11 @@ public JSONObject dispatch(JsonRpcPeer peer, String methodName, @Nullable JSONOb
           "Not implemented: " + methodName,
           null /* data */));
     }
+
+    Stetho.getAnalyticsLogger().track(
```

Comment:
I'm not a fan of how this method allocates even when there is no analytics logger present.  The default strategy you went with here is strange given that it just sends the data to logcat but this would be seen as useless noise to most of our users, even internally when there is no real analytics logger hooked up.  For these reasons, prefer an if-guard to be explicit and optimal:

```java
if (Stetho.hasAnalyticsLogger()) {
  Stetho.getAnalyticsLogger().track(...);
}
```

However I'd hold off on implementing this suggestion until we settle on the API as I am generally not a fan of having this as a global singleton.